src/main/java/com/esri/samples/map/open_mobile_map_package/OpenMobileMapPackageSample.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.map.open_mobile_map_package;
+
+import java.io.File;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.MobileMapPackage;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class OpenMobileMapPackageSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Open Mobile Map Package Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map view
+      mapView = new MapView();
+
+      //load a mobile map package
+      final String mmpkPath = new File("samples-data/mmpk/Yellowstone.mmpk").getAbsolutePath();
+      MobileMapPackage mobileMapPackage = new MobileMapPackage(mmpkPath);
+
+      mobileMapPackage.loadAsync();
+      mobileMapPackage.addDoneLoadingListener(() -> {
+        if (mobileMapPackage.getLoadStatus() == LoadStatus.LOADED && mobileMapPackage.getMaps().size() > 0) {
+          //add the map from the mobile map package to the map view
+          mapView.setMap(mobileMapPackage.getMaps().get(0));
+        } else {
+          Alert alert = new Alert(Alert.AlertType.ERROR, "Failed to load the mobile map package");
+          alert.show();
+        }
+      });
+
+      // add the map view to stack pane
+      stackPane.getChildren().addAll(mapView);
```

Comment:
Just an `add()` is needed, but that being picky. :)

Reply:
picky is good